pkg/kubeapiserver/options/authentication.go

```diff
@@ -503,6 +504,8 @@ func (o *BuiltInAuthenticationOptions) ApplyTo(authInfo *genericapiserver.Authen
 		authenticatorConfig.CustomDial = egressDialer
 	}
 
+	authenticatorConfig.CustomDial = newWebhookDialer(serviceResolver, nil)
```

Comment:
is ok to override the `egressDialer` here?

Reply:
thanks for the review, my mistake here
`egressDialer` should be used for `newWebhookDialer` as  a `baseDialer`